### PR TITLE
Feature/add-admin

### PR DESCRIPTION
**Pull request by aroon0808**
added admin attribute, 
source https://github.com/plataformatec/devise/wiki/How-To:-Add-an-Admin-Role

added admin view

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -7,9 +7,10 @@ class ApplicationController < ActionController::Base
   protected
 
   def logged_and_no_logged
-    user_signed_in? ? "logged" : "no_logged"
+    user_signed_in? ? current_user.admin? ?  "admin" : "logged" : "no_logged"
```
Comment (hirokiraj) [inline app/controllers/application_controller.rb]: extract to two methods, either in helper or in model

File: app/views/layouts/admin.html.erb [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+<html>
+  <head>
+    <title>Pik</title>
+    <%= csrf_meta_tags %> <%= csp_meta_tag %>
+    <%=stylesheet_link_tag'application', media: 'all' %>
+    <%=javascript_include_tag 'application' %>
+  </head>
+
+  <body>
+    <% flash.each do |key, value| %>
```
Comment (hirokiraj) [inline app/views/layouts/admin.html.erb]: maybe it can be delegated to partial

File: db/schema.rb [hunk 3]
```diff
@@ -34,6 +40,7 @@
   create_table "projects_users", force: :cascade do |t|
     t.bigint "user_id"
     t.bigint "project_id"
+    t.boolean "isActive", default: true
```
Comment (hirokiraj) [inline db/schema.rb]: why is it added to schema if there is no migration for it?

File: app/views/layouts/admin.html.erb (hunk 2 above)
Comment (aroon0808) [reply app/views/layouts/admin.html.erb]: i will do it later

**Pull request merged**